### PR TITLE
Added Click Function to Load More and Modal

### PR DESCRIPTION
**Pull request by jaxpham**

### CODE REVIEW

File: client/src/components/RatingsAndReviews/AddReview.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components';
+
+const StyledInput = styled.input`
+font-family: 'Helvetica', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif, Helvetica, sans-serif;
```
Comment (SimNeyzi) [inline client/src/components/RatingsAndReviews/AddReview.jsx:6]: Later you can refactor to create a global style for the font family. Just a suggestion :)

File: client/src/components/RatingsAndReviews/Reviews.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ const Reviews = ({ reviews, loading }) => {
   if (loading) {
     return <h2>Loading...</h2>;
   }
-  console.log(reviews);
+  // console.log(reviews);
```
Comment (SimNeyzi) [inline client/src/components/RatingsAndReviews/Reviews.jsx]: you can remove this?

File: client/src/components/RatingsAndReviews/ReviewsList.jsx [hunk 3]
```diff
@@ -1,50 +1,66 @@
...
-    //   setLoading(false);
-    // };
-    // fetchPosts();
     setLoading(true);
     axios('https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/reviews/', {
       headers: config,
       params: { product_id: productID },
     })
       .then((list) => {
+        console.log(list);
```
Comment (SimNeyzi) [inline client/src/components/RatingsAndReviews/ReviewsList.jsx]: console.log 🚨

**Review by SimNeyzi — APPROVED** [review]

**Comment by SimNeyzi** [comment]
@jaxpham - Looking good! Could you look into the conflicts when you get a chance?

File: client/src/components/RatingsAndReviews/AddReview.jsx (hunk 1 above)
Comment (jaxpham) [reply client/src/components/RatingsAndReviews/AddReview.jsx:6]: Yeah planning to do so next

**Pull request merged**